doc-content/jbpm-docs/src/main/asciidoc/Integration/ejb-interfaces-remote-ref.adoc

```diff
@@ -13,7 +13,7 @@ You can use these interfaces in the same way as local interfaces, with the excep
 
 You can define custom types in two ways. _Globally_ defined types are available on application classpath and included in the enterprise application. If you define a type _locally to the deployment unit_, the type is declared in a project dependency (for example, in a KJAR file) and is resolved at deployment time.
 
-Globally-available types do not require any special handling. The EJB container automatically marshalls the data when handling remote requests. However, local custom types are not visible to the EJB container by default. 
+Globally available types do not require any special handling. The EJB container automatically marshalls the data when handling remote requests. However, local custom types are not visible to the EJB container by default. 
```

Comment:
Actually I think the hypen is OK. We have two adjectives that modify the same noun. But you can leave the correction.